server.go

```diff
@@ -726,7 +726,7 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 		// this amount. If not, we return a 0 bandwidth because the
 		// link is unusable for this htlc.
 		if amount != nil {
-			err := link.MayAddOutgoingHtlc()
+			err := link.MayAddOutgoingHtlc(*amount)
```

Comment:
Ah I see where it slots in now!

Reply:
Just wanted to break up the commits here!